plugins/modules/vmware_guest_controller.py

```diff
@@ -216,7 +225,8 @@
                 "controller_disks_devicekey": [],
                 "controller_label": "SCSI controller 1",
                 "controller_summary": "VMware paravirtual SCSI",
-                "controller_unitnumber": 4
+                "controller_unitnumber": 4,
+                "controller_sharedbus": 'physicalSharing'
```

Comment:
```suggestion
                "bus_sharing": 'physicalSharing'
```

If I see this correct, the parameter is `bus_sharing` and not `controller_sharedbus`.

Reply:
as above